### DISCUSSION TITLE
1.1-es tesztkörnyezet

### DISCUSSION THREAD

**Discussion by kmolitorisz** [nav-gov-hu/M2M#40] [Kérdések]
Sziasztok!
Tudja valaki használni az új 1.1-es környezteben a tesztrendszert? Az ÜPO-n a tesztkörnyezet regisztrációnál található Kliens azonosító,Kliens jelszó,Felhasználónév,Felhasználó jelszó jó, meg is kapom a tesztkörnyezetben a tokent, a redeemNonce viszont már nem megy, az kapom vissza, hogy üres nonce. Lehet valahogy ezeket az azonosítókat újra generálni? Működik valakinek?

**Comment by Almito420** [comment]
Szia, az Aláírókulcs első fele és a Aláírókulcs második felének kiváltásához szükséges kód fel vannak cserélve a honlapon. Javítani fogják majd 
a [discussion](https://github.com/nav-gov-hu/M2M/discussions/37) alapján. Ha felcserélem, akkor én is megkapom a kulcs második felét.

**Reply by kmolitorisz** [reply]
Szuper, köszi, működik.

**Comment by rrd108** [comment]
Nem tudom, hogy a honlapon még mindig fel van-e a két adat cserélve, de nálam mindkettőre `{"resultMessage":"Nonce is empty","signatureKeySecondPart":null,"resultCode":"INVALID_NONCE"}` válasz jön.

Mi lehet vajon az oka?

A header:
```
"Accept: application/json",
"Content-Type: application/json",
"messageId: a491988a-7963-42e5-a0b2-2ce3f9103592",
"Authorization: Bearer TOKEN_AZ_APIRÓL"
```

Request body: `{"requestData":{"nonce":"UPO_TEST_NONCE"}}`

**Reply by rrd108** [reply]
A teszt környezetben van valami gubanc, ugyanis a ód az éles környezetben szépen bereggelte a usert.

**Comment by TamasPill** [comment]
Igen, ez most egy speciális probléma, ami ennél a usernél jelentkezett.
Hétfőn reseteljük a qnfr1hmd5n adatait és akkor újra elvégezhető lesz a tesztkörnyezet regisztráció.

Egyébként az ÜPO képernyőn már jól jelennek meg a teszkötnyezeti adatok.

**Comment by TamasPill** [comment]
Kedves rrd108! Reseteltük a tesztfelhasználót, így az ÜPO-n újra elvégezhető a tesztkörnyezet regisztráció. Az új felhasználóval már jól fog működni. Köszönjük a jelzést!

**Reply by rrd108** [reply]
köszi
